### DISCUSSION TITLE
Manual Level Z-Position Does Not Appear To Function

### DISCUSSION THREAD

**Discussion by Cal-LF** [Jyers/Marlin#400] [Q&A] [answered]
Making the adjustment to Z Position in Manual Level does not change the Z height. I tried setting Live Z Adjust.
What is the purpose of the this setting?

**Comment by Jyers** [comment, marked as answer]
The Z position item changes the Z position that the printer goes to at each point, it is not applied until you go to a point after setting it. Ideally you would set it to the approximate thickness of your probe so that z=0 is at the bed. However this is not necessary for everyone depending on how you handle leveling and such.
